template.go

```diff
@@ -54,15 +64,51 @@ func readYamlRequest(opts *Options) error {
 		return err
 	}
 
-	headers, err := yaml.Marshal(t.Headers)
-	if err != nil {
-		return err
+	if t.Peer != "" {
+		opts.TOpts.HostPorts = []string{t.Peer}
+	} else if t.Peers != nil {
+		opts.TOpts.HostPorts = t.Peers
+	} else if t.PeerList != "" {
+		opts.TOpts.HostPortFile = path.Join(path.Dir(opts.ROpts.YamlTemplate), t.PeerList)
```

Comment:
So unfortunately the path.Join doesn't work the way I was expecting, so we'll probably need to do is `IsAbs` check. We also shoudl use `filepath` instead of `path`, since `filepath` is OS specific, while `path` just assumes `/` (for those crazy systems where there's other separators)